internal/resources/policy/kind/mutation/recipe/label_schema.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+// nolint: dupl
+package recipe
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	policyrecipecustomcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/custom/common"
+	policyrecipemutationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/mutation"
+	policyrecipemutationcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/mutation/common"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/common"
+)
+
+var LabelSchema = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "The input schema for custom policy tmc_block_nodeport_service recipe version v1",
+	Optional:    true,
+	ForceNew:    true,
+	MaxItems:    1,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			targetKubernetesResourcesKey: common.TargetKubernetesResourcesSchema,
+			scopeKey: {
+				Type:         schema.TypeString,
+				Description:  "Scope",
+				Optional:     true,
+				Default:      "*",
+				ValidateFunc: validation.StringInSlice([]string{"*", "Cluster", "Namespaced"}, false),
+			},
+			LabelKey: {
+				Type:     schema.TypeList,
```

Comment:
Can `LabelKey` be a map instead of a list?

Reply:
I tried at some point but it was not suitable for this case and previously I got following error:
* resource tanzu-mission-control_mutation_policy: label: TypeMap with Elem *Resource not supported,use TypeList/TypeSet with Elem *Resource or TypeMap with Elem *Schema